lisa/environment.py

```diff
@@ -323,6 +324,21 @@ def create_node_from_requirement(
 
         return node
 
+    def local_node(self) -> Node:
```

Comment:
can it be a method like quick_connect at the bottom? It doesn't relate to the environment. The cached local node can be put in the module level too.

Reply:
It is worth noting that I also need access to the local node in some tests I am writing (in a private repo). However, I ran into an issue where the name of the two local nodes was the same and therefore the second node instance failed to initialize complaining that the log directory already existed. Hence, why I wanted to place the local node in a shared location that both places could access.

In regards to AZURE and READY not requiring local node (at the moment), this is why I lazy initialized the local node object.